distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestPlatform.java

```diff
@@ -434,24 +438,95 @@ public void testConfigurationFileWithInvalidFormat() throws IOException {
     }
 
     @Test
-    public void testExport() throws ConfigurationFileException {
-        String exportedDirectory = String.format("%s/etc/exported", ddfHome);
+    public void testExport() throws ConfigurationFileException, IOException {
         console.runCommand(EXPORT_COMMAND);
-        File[] exportedFiles = new File(exportedDirectory).listFiles();
+        File[] exportedFiles = new File(getExportDirectory()).listFiles();
         assertThat("Exported files should not be null.", exportedFiles, is(notNullValue()));
-        assertThat(String.format("No files exported to %s.", exportedDirectory),
+        assertThat(String.format("No files exported to %s.", getExportDirectory()),
                 Arrays.asList(exportedFiles), is(not(empty())));
+        FileUtils.deleteDirectory(new File(getExportDirectory()));
     }
 
     @Test
     public void testExportOnTopOfFile() throws ConfigurationFileException, IOException {
-        String exportedDirectory = String.format("%s/etc/exported", ddfHome);
-        File file = new File(exportedDirectory);
+        File file = new File(getExportDirectory());
         file.createNewFile();
-        assertThat(String.format("Should not have been able to export to %s.", exportedDirectory),
-                console.runCommand(EXPORT_COMMAND), containsString(
-                        String.format("Failed to export all configurations to %s",
-                                exportedDirectory)));
+        String response = console.runCommand(EXPORT_COMMAND);
+        assertThat(String.format("Should not have been able to export to %s.", getExportDirectory()),
+                response, containsString(String.format("Failed to export all configurations to %s",
+                        getExportDirectory())));
         file.delete();
     }
+
+    /*
+        Different Scenarios we want to test.
+        1.  testExport --> should check to see if there are the correct files
+        2.  testExportOnTopOfFile --> should check to see if the correct error is thrown
+        3.  testExport after saving a new configuration --> check to see if it is included in the export
+        4.  testExport after deleting a configuration --> check to make sure it isn't included in the export
+        5.  testImport after exporting?
+        6.
+     */
+
+    @Test
+    public void textExportAfterSavingAConfiguration() throws Exception {
+        managedServiceNewConfig1.addConfigurationFileAndWait(configAdmin);
+        console.runCommand(EXPORT_COMMAND);
+
+        assertThat("Saved configuration should be exported.",
+                new File(getPathToExportedConfig(managedServiceNewConfig1.pid)).isFile(),
+                is(true));
+        FileUtils.deleteDirectory(new File(getExportDirectory()));
```

Comment:
all the deletes have been swapped to the top for the methods that need them for the next commit